docs/blockentities/ber.md

```diff
@@ -1,44 +1,122 @@
+In the event that you do not need the BER provider context in your BER, you can also remove the constructor:
 
-## Registering a BER
+```java
+public class MyBlockEntityRenderer implements BlockEntityRenderer<MyBlockEntity> {
+    @Override
+    public void render( /* ... */ ) { /* ... */ }
+}
 
-In order to register a BER, you must subscribe to the `EntityRenderersEvent.RegisterRenderers` [event on the mod event bus][event] and call `#registerBlockEntityRenderer`.
+// In your event handler class
+@SubscribeEvent
+public static void registerEntityRenderers(EntityRenderersEvent.RegisterRenderers event) {
+    event.registerBlockEntityRenderer(MyBlockEntities.MY_BLOCK_ENTITY.get(),
+            // Pass the context to an empty (default) constructor call
+            context -> new MyBlockEntityRenderer()
+    );
+}
+```
 
-```java
-public class MyBlockEntityRenderer implements BlockEntityRenderer {
+## `BlockEntityWithoutLevelRenderer`
 
-    public MyBlockEntityRenderer(BlockEntityRendererProvider.Context ctx) {
-        // Do things here
+`BlockEntityWithoutLevelRenderer`, colloquially known as BEWLR, is an adaptation of the regular `BlockEntityRenderer` for special [item] rendering (hence "without level", as items do not have level context). Its overall purpose is the same: do special rendering for cases where static models aren't enough.
+
+To add a BEWLR, create a class that extends `BlockEntityWithoutLevelRenderer` and overrides `#renderByItem`. It also requires some additional constructor setup:
+
+```java
+public class MyBlockEntityWithoutLevelRenderer extends BlockEntityWithoutLevelRenderer {
+    // We need some boilerplate in the constructor, telling the superclass where to find
+    // the central block entity and entity renderers.
+    public MyBlockEntityWithoutLevelRenderer() {
+        super(Minecraft.getInstance().getBlockEntityRenderDispatcher(), Minecraft.getInstance().getEntityModels());
+    }
+    
+    @Override
+    public void renderByItem(ItemStack stack, ItemDisplayContext transform, PoseStack poseStack, MultiBufferSource bufferSource, int packedLight, int packedOverlay) {
+        // Do the rendering here.
     }
+}
+```
+
+Keep in mind that, like with BERs, there is only one instance of your BEWLR. Stack-specific properties should therefore be stored in the stack, not the BEWLR.
 
-    // Implement #render method here and any other logic
+Unlike BERs, we do not register BEWLRs directly. Instead, we register an instance of `IClientItemExtensions` to the `RegisterClientExtensionsEvent`. `IClientItemExtensions` allows us to specify a number of rendering-related behaviors on items, but since we're only interested in replacing the renderer with our newly-made BEWLR, we will just slap the `IClientItemExtensions` interface onto our BEWLR, like so:
+
+```java
+public class MyBlockEntityWithoutLevelRenderer extends BlockEntityWithoutLevelRenderer implements IClientItemExtensions {
```

Comment:
While this does work just fine, I don't think we should be suggesting this in the docs. Proper separation of concerns should be preferred in docs aimed mostly at beginners.